### PR TITLE
✨ source google ads: add full refresh custom table "asset_group_listing_group_filter"

### PR DESCRIPTION
**Pull request by jpmmcneill**
## What
Add a new table [asset_group_listing_group_filter](https://developers.google.com/google-ads/api/fields/v12/asset_group_listing_group_filter) for google ads full refresh custom tables

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Connector version is set to `0.0.1`
    - [ ] `Dockerfile` has version `0.0.1`
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] `docs/integrations/<source or destination>/<name>.md` including changelog with an entry for the initial version. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### CODE REVIEW

**Comment by marcosmarxm** [comment]
Thanks for the request @jpmmcneill I asked to @grubberr to take a look

**Comment by marcosmarxm** [comment]
@sh4sh please add one unit test to validate the cahnge.

**Comment by sh4sh** [comment]
@marcosmarxm could you explain which additional unit tests would add value here? There are already unit tests for custom tables added in https://github.com/airbytehq/airbyte/pull/22703

File: airbyte-integrations/connectors/source-google-ads/source_google_ads/source.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
 )
 from .utils import GAQL
 
-FULL_REFRESH_CUSTOM_TABLE = ["geo_target_constant", "custom_audience"]
+FULL_REFRESH_CUSTOM_TABLE = ["geo_target_constant", "custom_audience", "asset_group_listing_group_filter"]
```
Comment (lazebnyi) [inline airbyte-integrations/connectors/source-google-ads/source_google_ads/source.py:40]: Need bump version and update change log.

**Review by lazebnyi — DISMISSED** [review]

**Review by sh4sh — APPROVED** [review]

**Pull request merged**